WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -1,257 +1,293 @@
-          <a href="" class="dropdown-toggle top-nav" data-toggle="dropdown" data-hover="dropdown">
-            <%= ui_icon('list ol') %>
-            <span class="hidden-sm"><%= t '.results' %></span>
+  <div class="brand">
+    <a href="<%= root_path %>" ><%= image_tag "WCA Logo 2020.svg" %></a>
+  </div>
+  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+  </button>
+  <div class="navbar-collapse collapse disabled">
+    <ul class="nav navbar-nav">
+      <li class="top-nav logo">
+        <a href="<%= root_path %>"><%= image_tag "WCA Logo 2020.svg" %></a>
+      </li>
+      <li class="dropdown <%= Set['static_pages', 'posts'].include?(params[:controller]) ? '' : '' %>">
+        <a href="/" class="dropdown-toggle top-nav" data-toggle="dropdown" data-hover="dropdown">
+          <%= ui_icon('info circle') %> <span class="hidden-sm hidden-md"><%= t '.information' %></span> <span class="caret"></span>
+        </a>
+        <ul class="dropdown-menu" role="menu">
+          <li><a href="<%= about_path %>"><%= ui_icon('file alternate') %> <%= t '.about' %></a></li>
+          <li><a href="<%= documents_path %>"><%= ui_icon('copy') %> <%= t '.documents' %></a></li>
+          <li><a href="<%= delegates_path %>"><%= ui_icon('sitemap') %> <%= t '.delegates' %></a></li>
+          <li><a href="<%= teams_committees_path %>"><%= ui_icon('id card') %> <%= t '.teams' %></a></li>
+          <li><a href="<%= organizations_path %>"><%= ui_icon('flag') %> <%= t '.organizations' %></a></li>
+          <li class="divider"></li>
+          <li><a href="<%= faq_path %>"><%= ui_icon('question circle') %> <%= t '.faq' %></a></li>
+          <li><a href="<%= education_path %>"><%= ui_icon('graduation cap') %> <%= t '.education' %></a></li>
+          <li><a href="<%= contact_website_path %>"><%= ui_icon('envelope') %> <%= t '.contact' %></a></li>
+          <li><a href="<%= media_instagram_path %>"><%= ui_icon('camera') %> <%= t '.media_submission' %></a></li>
+          <li><a href="<%= privacy_path %>"><%= ui_icon('user secret') %> Privacy</a></li>
+          <li><a href="<%= disclaimer_path %>"><%= ui_icon('exclamation circle') %> Disclaimer</a></li>
+          <li class="divider"></li>
+          <li><a href="<%= score_tools_path %>"><%= ui_icon('wrench') %> <%= t '.tools' %></a></li>
+          <li><a href="<%= logo_path %>"><i><%= image_tag('wca_logo.svg', width: '16px') %></i> <%= t '.logo' %></a></li>
+        </ul>
+      </li>
+      <li class="dropdown <%= Set['competitions', 'delegate_reports', 'results_submission'].include?(params[:controller]) ? 'active' : '' %>">
+        <a href="" class="dropdown-toggle top-nav" data-toggle="dropdown" data-hover="dropdown">
+            <%= ui_icon('cube') %><span class="hidden-sm hidden-md"> <%= t '.competitions' %></span> 
```

Comment:
Nit: the build fails because of the trailing newline here :>

Reply:
Sorry, I'm not too sure where the trailing newline is, and I'm not exactly sure what it is either. It is just a "\n" character?

Reply:
Whoops, I meant trailing whitespace, basically a space (`" "`) at the end of the line.

Reply:
Got it. For future reference, how can I tell why the build fails?

Reply:
You can click on `Details` here:

![image](https://user-images.githubusercontent.com/17034772/101287615-99af9880-37f1-11eb-8860-26f42fc89504.png)

And then on `The build`:

![image](https://user-images.githubusercontent.com/17034772/101287626-a207d380-37f1-11eb-85cb-541b63b39a57.png)